serverless/knative_eventing/serverless-using-brokers.adoc

```diff
@@ -6,11 +6,44 @@ include::modules/serverless-document-attributes.adoc[]
 
 toc::[]
 
-Knative Eventing uses the `default` broker unless otherwise specified.
+Knative provides a multi-tenant (MT) Broker implementation that uses Channels for event routing.
```

Comment:
The MT broker isn't described well. What the difference on MT broker and regular broker? Writing this in this way may confuse newcomers.

The problem is that the MT broker would be a new default broker in Serverless 1.8.0, and the old broker will be removed. Thus, it will be called simply "broker".

Now, we are in transition phase, and have this old broker as a default, and MT as new one.

The difference between MT broker and old regular one is that, old broker when was created for given namespace, have created 2 additional pods in that namespace. So, the more old brokers there ware created, more pods (resources) ware used. MT broker don't do that, as he only create pods in knative-eventing system namespace, and there number of MT broker pods is static, is not dependent on number of brokers.

Reply:
Thank you, this is really helpful information. I think maybe we need to fix this upstream first before it makes sense to document properly. This is on my to do list, but makes a lot more sense now how to fix it with this explanation.